pepper-apis/docs/specification/src/components/schemas.yml

```diff
@@ -1724,12 +1754,31 @@ Question.Composite:
       additionalItemText:
         type: string
         description: 'The label to show above entries beyond the initial one'
+      childVisibility:
```

Comment:
That `childVisibility` array should have same length and order of items like `children` property, right?

Reply:
The format for the array is based off of how we're handling the `CompositeAnswer` values. The top-level array will either have 1 entry (the case where the questions need to be presented, but the user has interacted with the composite question yet), or a number of elements equal to the number of answers. The lower-level array will have the same length and order.

I've added a couple other formats I had considered below- I'm also open to suggestions if you have an alternate representation that may offer other tradeoffs.

## Examples
_any values for shown/enabled below are placeholders_

### 2 questions, no answers
```json5
[
  [
    { // Question 0
      "shown": true,
      "enabled": true
    },
    { // Question 1
      "shown": true,
      "enabled": true
    }
  ]
]
```

### 2 questions, 3 answers
```json5
[
  [
    { // Question 0, answer 0
      "shown": true,
      "enabled": true
    },
    { // Question 1, answer 0
      "shown": true,
      "enabled": true
    }
  ],
  [
    { // Question 0, answer 1
      "shown": false,
      "enabled": true
    },
    { // Question 1, answer 1
      "shown": true,
      "enabled": false
    }
  ],
  [
    { // Question 0, answer 2
      "shown": true,
      "enabled": false
    },
    { // Question 1, answer 2
      "shown": false,
      "enabled": true
    }
  ]
]
```

Reply:
That looks good for me. A benefit is the backward compatibility as well.